ckanext/recombinant/logic.py

```diff
@@ -259,7 +259,8 @@ def _dataset_fields(geno):
     """
     return the dataset metadata fields created for dataset definition geno
     """
-    return {'title': geno['title'], 'notes': geno.get('notes', '')}
+    notes = geno.get('alt_notes', geno.get('notes', ''))
+    return {'title': geno['title'], 'notes': notes}
```

Comment:
this seems strange. Why replace the notes with alt_notes?

This field gets stored on the dataset itself and will appear in the dataset search page on the registry. The text you've used in ckanext-canada doesn't seem appropriate.